src/stories/components/Input.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {useState} from "react";
+import '../css/input.css';
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Input = ({ variant, label, placeholder, leftIcon, rightIcon, isDisabled, rows, cols, error, success, onChange, ...props }) => {
+  const [input, setInput] = useState(null);
+  const handleChange =(newValue)=>{    
+    setInput(newValue);
+    console.log(newValue); 
+  }
+  return (  
+    <div className={'storybook-input-div'}>
+      <label className={'storybook-input-label'}>{label}</label>
+      {variant === 'textArea'? 
+      (<textarea
+        className={['storybook-input','storybook-input--' + (isDisabled? 'disabled':''),'storybook-input--' + (error?'error':''), 'storybook-input--' + (success?'success':''),`storybook-input--${variant}`].join(' ')}
+        onChange={handleChange}
+        placeholder={placeholder}
+        disabled={isDisabled}
+        rows={rows}
+        cols={cols}
+        {...props}
+      />):
+      (
+      <div 
+      className={['storybook-input','storybook-input--' + (isDisabled? 'isDisabled':''),'storybook-input--' + (error?'error':''), 'storybook-input--' + (success?'success':''), `storybook-input--${variant}`].join(' ')}
+      >
+        {(variant === "leftIcon" ||  variant==="doubleIcon" ) && leftIcon ? <img className='input-icon--left' src={leftIcon}></img> : ''}
+        <input
+          type="text"
+          onChange={handleChange}
+          placeholder={placeholder}
+          disabled={isDisabled}
+          {...props}
+        />
+        {(variant === "rightIcon" ||  variant==="doubleIcon" ) && rightIcon ? <img className='input-icon--right' src={rightIcon}></img> : ''}
+      </div>
+      )}
+      {error === true && 
```

Comment:
NIT: Change this line to `error &&` please. Since we are (unofficially) trying to stay consistent with ES6 standards, this is a much cleaner way to write this code!

Reply:
done